int32[] damage
geometry_msgs/Point[] location
---
geometry_msgs/Point[] creeper_locations
